.github/workflows/ceqr_dep_monthly.yml

```diff
@@ -3,32 +3,64 @@ on:
   workflow_dispatch:
   schedule:
     - cron: 0 0 1 * *
-  # repository_dispatch:
-  #   types: [dep_cats_permits]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        shell: bash
+        working-directory: products/ceqr
+    container:
+      image: nycplanning/build-geosupport:${{ inputs.image_tag || 'latest' }}
     env:
-      RECIPE_ENGINE: ${{ secrets.RECIPE_ENGINE }}
-      EDM_DATA: ${{ secrets.EDM_DATA }}
-      AWS_S3_ENDPOINT: ${{ secrets.AWS_S3_ENDPOINT }}
-      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      BUILD_ENGINE_DB: defaultdb
     steps:
-      - uses: actions/checkout@v2
-        
-      - name: Install dependencies
-        shell: bash
-        run: ./ceqr setup
-        
+      - uses: actions/checkout@v4
+
+      - name: Load Secrets
+        uses: 1password/load-secrets-action@v1
+        with:
+          export-env: true
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+          BUILD_ENGINE_SERVER: "op://Data Engineering/EDM_DATA/server_url"
+          AWS_S3_ENDPOINT: "op://Data Engineering/DO_keys/AWS_S3_ENDPOINT"
+          AWS_SECRET_ACCESS_KEY: "op://Data Engineering/DO_keys/AWS_SECRET_ACCESS_KEY"
+          AWS_ACCESS_KEY_ID: "op://Data Engineering/DO_keys/AWS_ACCESS_KEY_ID"
+
+      - name: Setup build environment
+        working-directory: ./
+        run: |
+          ./bash/docker_container_setup.sh
+
       - name: Run recipe
-        shell: bash
-        run: ./ceqr run recipe dep_cats_permits
-  
-      - uses: JasonEtco/create-an-issue@v2
-        name: Create Issue to Publish
+        run: |
+          export EDM_DATA=$BUILD_ENGINE_SERVER/$BUILD_ENGINE_DB
+          python3 -m pip install mdpdf
+          ./ceqr run recipe dep_cats_permits
+
+      - name: Create issue on success
+        if: success()
+        uses: JasonEtco/create-an-issue@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          filename: .github/ISSUE_TEMPLATE/dep-monthly.md
+
+  create_issue_on_failure:
```

Comment:
Just curious why it's a separate job, rather than a step (like the `Create issue on success` step above)?

Reply:
good call! no good reason that I can think of, will change

Reply:
I noticed that all uses of `scheduled_action_failure.md` are in a separate job, not sure why but maybe it's a quirk of the clause `if: ${{ failure() && (github.event_name == 'schedule') }}` that @fvankrieken bumped into?

Reply:
Not that I can remember!